lib/internal/test_runner/reporter/tap.js

```diff
@@ -3,6 +3,7 @@ const {
   ArrayPrototypeForEach,
   ArrayPrototypeJoin,
   ArrayPrototypePush,
+  Date,
```

Comment:
You know you can do suggestions on your own PRs, right?
```suggestion
```

Reply:
Did not know that. If I start a review? Will have to read up on that. The user interface at least is not particularly intuitive for add your own suggestion to the current commit batch.